index.js

```diff
@@ -0,0 +1,64 @@
+const dotenv = require('dotenv')
+const { Bot } = require('grammy');
+
+dotenv.config();
+
+const bot = new Bot(process.env.BOT_TOKEN);
+
+bot.use((ctx, next) => {
+    ctx.state = {};
+    return next();
+})
+
+const middlewareMatchToSplit = function (ctx, next){
+    if(!ctx.match){
+        return ctx.reply("Please enter a number")
+    }
+    ctx.state.arguments = ctx.match.split(' ');
+    const validateNumbers = ctx.state.arguments.some((elem) => {
+        return Number.isNaN(+(elem))
+    })
+
+    if(validateNumbers){
+        return ctx.reply("Incorrect number entered")
+    }
```

Comment:
No additional space after the block ends

Reply:
Done